tests/OnlineSales.Tests/MediaTests.cs

```diff
@@ -21,6 +22,22 @@ public async Task CreateAndGetMediaTest(string fileName, int fileSize, bool shou
         result.Should().Be(shouldBePositive);
     }
 
+    [Theory]
+    [InlineData("Привет, мир 3120.png", "privetmir3120.png", 1024)]
```

Comment:
What happens if we will use Chinese, Italian, Sinhala language?

Reply:
There's no support to Asian, Latin languages, only Cyrillic -> Latin and Latin with diacrytics -> Latin